src/System.Private.ServiceModel/tests/Scenarios/Binding/Http/BasicHttpBindingTests.cs

```diff
@@ -4,17 +4,20 @@
 using System;
 using System.ServiceModel;
 using System.ServiceModel.Channels;
+using System.ServiceModel.Tests.Common;
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 using TestTypes;
 using Xunit;
 
-public static class Binding_Http_BasicHttpBindingTests
+public class Binding_Http_BasicHttpBindingTests : OuterLoopTests
```

Comment:
Is there any other per test initialization strategy. One time initialization needs to happen somewhere. If these aren't available in native then that's an issue. The brigde would be initialized only one time but each test needs to Ping it. So if the Ping is the check then a static helper like GetOrCreate("servicename") might be all it takes.